src/blocks/sound.rs

```diff
@@ -177,48 +389,28 @@ impl ConfigBlock for Sound {
             step_width = 50;
         }
 
-        let sound = Sound {
+        let mut sound = Sound {
             text: ButtonWidget::new(config.clone(), &id).with_icon("volume_empty"),
             id: id.clone(),
-            devices: vec![SoundDevice::new("Master")?],
+            devices: vec![
+                // TODO: find better solution for mixed types
+                match PulseAudioSoundDevice::new(0) {
```

Comment:
It seems unlikely that users will have mixed types if they are opting in to PulseAudio for one. However, we clearly need a way of configuring the device in use here. I don't think we should be controlling the volume of multiple devices from the same block.